### PR TITLE
add my site

### PR DESCRIPTION
**Pull request by thecrewmate876**
cc @wgyt

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -8,4 +8,13 @@
     "user": "wgyt",
     "username": "William Horning"
   }
+  {
```
Comment (wgyt) [inline data.json:11]: Please add a comma, this is JSON.
```suggestion
  },
  {
```

Comment (thecrewmate876) [reply data.json:11]: oops i forgot

**Review by wgyt — CHANGES_REQUESTED** [review]
finish your damn website

**Comment by thecrewmate876** [comment]
> 
> 
> finish your damn website

@wgyt it's really just a blog and honestly idk what to put on the home page

**Comment by wgyt** [comment]
> > finish your damn website
> 
> @wgyt it's really just a blog and honestly idk what to put on the home page

add an about me or blog posts list there

**Comment by thecrewmate876** [comment]
> 
> 
> > > finish your damn website
> > 
> > 
> > @wgyt it's really just a blog and honestly idk what to put on the home page
> 
> add an about me or blog posts list there

@wgyt https://thecrewmate876.wordpress.com/blog-2

**Comment by wgyt** [comment]
> > > > finish your damn website
> > > 
> > > 
> > > @wgyt it's really just a blog and honestly idk what to put on the home page
> > 
> > 
> > add an about me or blog posts list there
> 
> @wgyt https://thecrewmate876.wordpress.com/blog-2

add the post list to the home page

**Pull request closed without merging**